### ISSUE TITLE
Hi Yami! 

### ISSUE THREAD

**Issue by michaelpfaff** [yamikuronue/SockLogiDice#12]
Was having issues with the dice roller, so I reinstalled it and now I can't even get it to load. Getting this error, which has set me back two steps. Ha! 

```
Plugin sock-logidice failed to load with error: TypeError: Cannot create property 'spoilers' on boolean 'true'
[2018-03-21T02:47:20.846Z] A-derp! Fatal Startup Error: Cannot create property 'spoilers' on boolean 'true'
```

Was wondering if you might know what's up. 

My configuration.yml file has it setup like this: 

```
plugins:
  sock-logidice: true
```

Is my configuration setup incorrectly?

**Comment by michaelpfaff** [comment]
Figured out how to get it to roll again! Now, just need to figure out why it's not rolling in threads and just chat.

**Comment by yamikuronue** [comment]
Oh good! What was the answer?

**Comment by michaelpfaff** [comment]
So, I had the configuration messed up. It should be...

```
sock-logidice: 
   spoilers: enabled
```

Without the "true". 

I also figured out what was up with it not rolling in threads, but working fine in chat. Because the bot is triggered on notification, it seems as if when I upgraded to v1.8.0 of NodeBB somehow the notifications got turned off for mentions / replies to threads being watched. When I turned those back on, bot worked fine. And, I face palmed as I went through a crazy amount of reinstalling software to try and fix the issue when it was actually... pretty simple. Argh. 

All is well now though. Thanks!
